src/core/gltf2importer/gltf2importer.h

```diff
@@ -65,11 +67,14 @@ class KUESASHARED_EXPORT GLTF2Importer : public Qt3DCore::QNode
     GLTF2Importer::Status status() const;
     Kuesa::SceneEntity *sceneEntity() const;
     bool assignNames() const;
+    Kuesa::GLTF2Import::GLTF2Options *options();
+    const Kuesa::GLTF2Import::GLTF2Options *options() const;
 
 public Q_SLOTS:
     void setSource(const QUrl &source);
     void setSceneEntity(Kuesa::SceneEntity *sceneEntity);
     void setAssignNames(bool assignNames);
+    void setOptions(const Kuesa::GLTF2Import::GLTF2Options &options);
```

Comment:
having a setter for a `CONSTANT` property seems weird. Why do you need it?
Isn't writing `importer->options->set...()` nicer?

Reply:
Was a decision to make it a bit more cppish. You can access directly the itnernal object as you suggest or use the setter, which just copies the value internally

Reply:
hm, superfluous I think, but ok. But the ownership rules are a bit weird...

If I call `setOptions(options)` and `options` has no parent, I would expect that the importer takes ownership of the options object. It of course doesn't because the data is copied. But it's still different from elsewhere in Qt3d/Kuesa

Reply:
Well, you are passing a const ref, no way to take ownership, as typical in c++. I can remove it if you prefer it.

Reply:
ah right, oops, read it as a pointer. I just find the API weird

Reply:
I think we can leave it as it is. We have the cpp and qml idioms.